src/NonBlocking/NonBlocking.csproj

```diff
@@ -39,7 +39,6 @@
   <PropertyGroup>
     <Nullable>enable</Nullable>
     <WarningsAsErrors>nullable</WarningsAsErrors>
-    <IsTrimmable>true</IsTrimmable>
```

Comment:
This is inconvenient. If NS2.0 or 2.1 does not care for the flag, why is it complaining? Is there any harm?

I wonder if this can be made conditional on the target TFM?

Reply:
It seems that with the current tooling it would require explicitly targeting `net6.0`: https://learn.microsoft.com/en-us/dotnet/core/tools/sdk-errors/netsdk1195 since `Condition` would not have any effect otherwise (neither `NS2.0` nor `NS2.1` are valid).

Reply:
The error sound like if what we have right now does not really have effect, although it is a bit confusing. I'll ask around if there is a way to be both trimmable and target NS.

Reply:
I think we should remove this for now, since it breaks the build.

Reply:
Thanks for looking into this